### ISSUE TITLE
Training error occurred when 'loss at 10000'

### ISSUE THREAD

**Issue by SceneLinker** [ymxlzgy/echoscene#10]
Hi thanks for the awesome work!
I faced an error while training your code.
I was training Echoscene with 250 epochs (NVIDIA A40 GPU) following the read me instruction. 
When I set the python, cuda, pytorch version all as instructed and train it, I get a RuntimeError at 10000th loss. 
I have tried to train again about 3 times, it works fine the first time, but at 10000th loss(124th epoch) the error is the same. I also tried reducing the batch size or worker size to solve it, but I encountered the same error.
I haven't modified the code or anything else at all. How can I fix this error? (btw, your similar model Commonscenes, trained and inferenced perfectly fine!)

![스크린샷 2024-10-22 오후 7 12 55](https://github.com/user-attachments/assets/1481c4c8-8f95-4807-a797-374aa958fe9e)

**Comment by ManateeSama** [comment]
same question，did u solve it？
